workspaces/app/src/containers/App/index.tsx

```diff
@@ -4,6 +4,7 @@ import CubeBlocksDecorative1Link from '@sepraisal/praisal/vendor/DecorativePack/
 import CubeBlocksDecorative2Link from '@sepraisal/praisal/vendor/DecorativePack2/CubeBlocks.sbc'
 import CubeBlocksEconomyLink from '@sepraisal/praisal/vendor/Economy/CubeBlocks.sbc'
 import CubeBlocksFrostbiteLink from '@sepraisal/praisal/vendor/Frostbite/CubeBlocks.sbc'
+import CubeBlocksSparksOfTheFutureLink from '@sepraisal/praisal/vendor/SparksOfTheFuturePack/CubeBlocks.sbc'
```

Comment:
add Symbols too

Reply:
Could you elaborate a bit what you mean by symbols? Where I come from it would mean to name the specific symbols you want to use from an import instead of cluttering the namespace with a complete import.

Reply:
Yes, sorry, it was written in a late hour :stuck_out_tongue: Here comes the elaborate version:

Sparks of the future introduced also a lot of letters in another file `CubeBlocks_Symbols`. It also should be added. Please do so here and elsewhere where appropriate.

```ts
import CubeBlocksSymbolsLink from '@sepraisal/praisal/vendor/Vanilla/CubeBlocks/CubeBlocks_Symbols.sbc'
```

P.S.  Which makes me wonder how it didn't work last night, or rather, worked only partially. Are we missing here some more files?

Reply:
Oh right, the base game files. Totally forgot about those! The only other file that may be related to blueprints is  the new LCD images in `Data/LCDTextures_SoF.sbc`.

Reply:
That explains! Please add it too